### PR TITLE
bump versions to v6.5.3

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Bump versions to v6.5.3

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/13871
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-cluster.md [hunk 1]
```diff
@@ -554,11 +554,11 @@ tiup cluster audit
 Starting component `cluster`: /home/tidb/.tiup/components/cluster/v1.11.3/cluster audit
 ID      Time                       Command
 --      ----                       -------
-4BLhr0  2023-04-21T13:25:09+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.2 /tmp/topology.yaml
-4BKWjF  2023-04-21T23:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.2 /tmp/topology.yaml
-4BKVwH  2023-04-21T23:02:08+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.2 /tmp/topology.yaml
+4BLhr0  2023-04-21T13:25:09+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.3 /tmp/topology.yaml
```
Comment (qiancai) [inline tiup/tiup-cluster.md:557]: 请修改下 L557~L561 的日期为今天

File: upgrade-tidb-using-tiup.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ title: 使用 TiUP 升级 TiDB
 
 查阅 TiDB release notes 中的兼容性变更。如果有任何变更影响到了你的升级，请采取相应的措施。
 
-以下为从 v6.4.0 升级至当前版本 (v6.5.2) 所需兼容性变更信息。如果从 v6.3.0 或之前版本升级到当前版本，可能也需要考虑和查看中间版本 [release notes](/releases/release-notes.md) 中提到的兼容性变更信息。
+以下为从 v6.4.0 升级至当前版本 (v6.5.3) 所需兼容性变更信息。如果从 v6.3.0 或之前版本升级到当前版本，可能也需要考虑和查看中间版本 [release notes](/releases/release-notes.md) 中提到的兼容性变更信息。
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:50]: L53 后面需要加 v6.5.3 release notes 的链接

File: ticdc/ticdc-avro-protocol.md [hunk 3]
```diff
@@ -282,4 +282,4 @@ Schema Registry 支持三种 [Subject Name Strategy](https://docs.confluent.io/p
 
 ## 兼容性说明
 
-* 在升级 TiCDC 集群到 v6.5.2 或更高的 v6.5.x 版本时，如果使用 Avro 同步的表包含 `FLOAT` 类型数据，请在升级前手动调整 Confluent Schema Registry 的兼容性策略为 `None`，使 changefeed 能够成功更新 schema。否则，在升级之后 changefeed 将无法更新 schema 并进入错误状态。详情请参考 GitHub issue [#8490](https://github.com/pingcap/tiflow/issues/8490)。
+* 在升级 TiCDC 集群到 v6.5.3 或更高的 v6.5.x 版本时，如果使用 Avro 同步的表包含 `FLOAT` 类型数据，请在升级前手动调整 Confluent Schema Registry 的兼容性策略为 `None`，使 changefeed 能够成功更新 schema。否则，在升级之后 changefeed 将无法更新 schema 并进入错误状态。详情请参考 GitHub issue [#8490](https://github.com/pingcap/tiflow/issues/8490)。
```
Comment (qiancai) [inline ticdc/ticdc-avro-protocol.md:285]:
```suggestion
* 在升级 TiCDC 集群到 v6.5.2 或更高的 v6.5.x 版本时，如果使用 Avro 同步的表包含 `FLOAT` 类型数据，请在升级前手动调整 Confluent Schema Registry 的兼容性策略为 `None`，使 changefeed 能够成功更新 schema。否则，在升级之后 changefeed 将无法更新 schema 并进入错误状态。详情请参考 GitHub issue [#8490](https://github.com/pingcap/tiflow/issues/8490)。
```

File: tiup/tiup-cluster.md [hunk 4]
```diff
@@ -554,11 +554,11 @@ tiup cluster audit
...
+4BKWjF  2023-04-21T23:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.3 /tmp/topology.yaml
+4BKVwH  2023-04-21T23:02:08+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.3 /tmp/topology.yaml
```
Comment (ran-huang) [inline tiup/tiup-cluster.md:559]:
```suggestion
4BLhr0  2023-06-14T13:25:09+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.3 /tmp/topology.yaml
4BKWjF  2023-06-14T23:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.3 /tmp/topology.yaml
4BKVwH  2023-06-14T23:02:08+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.3 /tmp/topology.yaml
```

File: tiup/tiup-cluster.md [hunk 5]
```diff
@@ -554,11 +554,11 @@ tiup cluster audit
...
 4BKKH1  2023-04-21T16:39:04+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster destroy test
```
Comment (ran-huang) [inline tiup/tiup-cluster.md:560]:
```suggestion
4BKKH1  2023-06-14T16:39:04+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster destroy test
```

File: tiup/tiup-cluster.md [hunk 6]
```diff
@@ -554,11 +554,11 @@ tiup cluster audit
...
-4BKKDx  2023-04-21T16:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.2 /tmp/topology.yaml
+4BKKDx  2023-04-21T16:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.3 /tmp/topology.yaml
```
Comment (ran-huang) [inline tiup/tiup-cluster.md:561]:
```suggestion
4BKKDx  2023-06-14T16:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.3 /tmp/topology.yaml
```

**Pull request merged**